### ISSUE TITLE
PSC configuration pop up appears after Admiral is already configured

### ISSUE THREAD

**Issue by aangelov-vmware** [vmware/vic-product#546]
I deployed VIC from OVA. Then I configured the PSC on  https://hostname:9443. Then Admiral was accessible on  https://hostname:8282. After using it for a while I went to   https://hostname:9443. The PSC configuration pop up was visible again. This does not seems correct because Admiral should be configured with PSC only once.

**Comment by stuclem** [comment]
I experienced the same thing, using chin-vic-0aa9d8da-dev.ova, and when trying to connect to the Management Portal on port 8282, I get a blank screen.

**Comment by andrewtchin** [comment]
Duplicate of https://github.com/vmware/vic-product/issues/484, closing
